lib/commands.js

```diff
@@ -2,34 +2,41 @@
 
 import { CompositeDisposable, Emitter } from 'atom'
 import type { Disposable } from 'atom'
-import type { State } from './types'
 
 export default class Commands {
-  state: State;
   emitter: Emitter;
   subscriptions: CompositeDisposable;
 
-  constructor(state: State) {
-    this.state = state
+  constructor() {
     this.emitter = new Emitter()
     this.subscriptions = new CompositeDisposable()
 
     this.subscriptions.add(this.emitter)
+    this.subscriptions.add(atom.commands.add('atom-workspace', {
+      'linter:debug': () => this.debug(),
```

Comment:
`debug()` needs a `TextEditor` instance, which it could be triggered without one from here, is there a reason it was moved?